src/ol/renderer/canvas/vectortilelayer.js

```diff
@@ -276,7 +276,7 @@ ol.renderer.canvas.VectorTileLayer.prototype.createReplayGroup = function(tile,
 /**
  * @inheritDoc
  */
-ol.renderer.canvas.VectorTileLayer.prototype.forEachFeatureAtCoordinate = function(coordinate, frameState, callback, thisArg) {
+ol.renderer.canvas.VectorTileLayer.prototype.forEachFeatureAtCoordinate = function(coordinate, frameState, hitTolerance, callback, thisArg) {
   var resolution = frameState.viewState.resolution;
   var rotation = frameState.viewState.rotation;
   var layer = this.getLayer();
```

Comment:
> Another question about the vectortilelayer for me is: how are features stored which are in multiple tiles? Could a simple buffering up the extent result in multiple calls of the callback for the same feature?

Yes, that could happen. We probably want to check that the callback is only called once for features with the same id?